text/refactor_refman.md

```diff
@@ -35,19 +37,14 @@ Some of the more apparent issues with the current documentation are:
      scattered across multiple sections.
 -	**Hard to locate concepts in the document.** For example, there is
      no subject index where the reader could look up unification.  The
-     HTML search box is not very helpful. (You can't see the
-     matching text for each search result without clicking on the
-     link.  Clicking on the link takes you to the top of the section
-     rather than to the matching text.)
+     HTML search box is not very helpful. (This is something that will
```

Comment:
I think it would be better to keep the original detail (below).  It is relevant to usability, although I don't expect we'll get to this soon and perhaps not at all.  This section is the motivation.  It's not a promise that we address every point.

(You can't see the
     matching text for each search result without clicking on the
     link.  Clicking on the link takes you to the top of the section
     rather than to the matching text.)